### ISSUE TITLE
Allow upload of Excel files

### ISSUE THREAD

**Issue by nealhaddaway** [ESHackathon/eviatlas#84]
Probably behavior: treat .xlsx files like .csv files (only reading in first sheet).

Add readxl to requirements
should be implement in same stage as moving CSV read from base R read.csv to tidy read_delim

**Comment by DrMattG** [comment]
I have managed to upload the excel file but it needs some work. At the moment I have to comment out the JS code in the dataTable (Map Database) to get it to render. I have no idea why! See code here: https://github.com/ESHackathon/eviatlas/blob/bfb46bbbf4364d8d5e150e9927fed4e6549eb665/server.R#L322

Have you got any ideas what is going wrong here (if you uncomment the JS section you should see a blank Map Database but the Atlas displays correctly)?

**Comment by nealhaddaway** [comment]
Yeah that works for me - but the commented out code doesn't seem to be impingeing on how it handles the xlsx file - everything seems to be working fine. I wonder what it did? Seems to be something to do with popups...

It works OK with the sample data as well, so it can't be the csv file...

Maybe we can ask Martin and Andrew?
________________________________

**Comment by DrMattG** [comment]
Yeah good idea to ask Andrew and Martin!
